addons/account/models/account_invoice.py

```diff
@@ -255,6 +255,7 @@ def _compute_payments(self):
     state = fields.Selection([
             ('draft','Draft'),
             ('open', 'Open'),
+            ('in_payment', 'In Payment'),
```

Comment:
Can be tricky, have you checked for place in code where we might have a domain that looks like ('state', '!=', 'open') or ('state', 'in', ('draft', 'open')) or things like that?